config/config.yml.example

```diff
@@ -1,107 +1,211 @@
-    # {
-    #   "Version": "2012-10-17",
-    #   "Statement": [
-    #     {
-    #       "Sid": "VisualEditor0",
-    #       "Effect": "Allow",
-    #       "Action": "s3:*",
-    #       "Resource": [
-    #         "arn:aws:s3:::bucket-name/*",
-    #         "arn:aws:s3:::bucket-name"
-    #       ]
-    #     }
-    #   ]
-    # }
-    endpoint:
-    asset_host:
-    access_key:
-    secret_key:
-    bucket:
-    bucket_region:
+    endpoint: 'http://minio:9000'
+    asset_host: 'http://localhost:9000/check-api-dev'
+    access_key: 'AKIAIOSFODNN7EXAMPLE'
+    secret_key: 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
+    bucket: 'check-api-dev'
+    bucket_region: 'us-east-1'
     path_style:
 
+  # Email settings
+  #
+  # REQUIRED for notification emails
+  #
+  support_email: 'check@meedan.com'
+  default_mail: 'Meedan Check <root@localhost>'
+  send_welcome_email_on_registration: false
+  smtp_host: 'smtp.gmail.com'
+  smtp_port: 587
+  smtp_user: '<GMAIL USERNAME>'
+  smtp_pass: '<GMAIL PASSWORD>'
+
+  # Pusher is used for push notifications from backend to frontend https://pusher.com/channels
+  #
+  # REQUIRED
+  #
+  pusher_app_id: '<PUSHER APP ID>'
+  pusher_key: '<PUSHER KEY>'
+  pusher_secret: '<PUSHER SECRET>'
+  pusher_cluster: '<PUSHER CLUSTER>'
+
+  # Transifex is used to localize strings https://transifex.com
+  #
+  # REQUIRED for tasks `rake transifex:XXX` and dynamic string localization
+  # By default, latest localizations are included in the codebase
+  #
+  # Offical Check localization at https://transifex.com/meedan/check-2
+  # The user should have a Transifex role of Project Maintainer
+  #
+  transifex_user: '<TRANSIFEX USERNAME>'
+  transifex_password: '<TRANSIFEX PASSWORD>'
+  transifex_project: 'check-2'
+
+  # Twitter social login
+  #
+  # REQUIRED for Twitter social login
+  #
+  twitter_consumer_key: '<TWITTER APP CONSUMER KEY>'
+  twitter_consumer_secret: '<TWITTER APP CONSUMER SECRET>'
+
+  # Facebook social login
+  #
+  # REQUIRED for Facebook social login
+  #
+  facebook_app_id: '<FACEBOOK APP ID>'
+  facebook_app_secret: '<FACEBOOK APP SECRET>'
+
+  # Slack social login
+  #
+  # REQUIRED for Slack social login
+  #
+  slack_app_id: '<SLACK APP ID>'
+  slack_app_secret: '<SLACK APP SECRET>'
+
+  # Google social login
+  #
+  # REQUIRED for Google social login
+  #
+  google_client_id: '<GOOGLE CLIENT ID>'
+  google_client_secret: '<GOOGLE CLIENT SECRET>'
+  google_auth_redirect_uri: 'http://localhost:3000/api/users/auth/google_oauth2/callback'
+  google_credentials_path: '/app/config/credentials.json'
+
+  # Smooch (Zendesk Sunshine Conversations) messaging platform gateway
+  #
+  # REQUIRED for Check Message
+  #
+  smooch_twitter_consumer_key: '<SMOOCH CONSUMER KEY>'
+  smooch_twitter_consumer_secret: '<SMOOCH CONSUMER SECRET>'
+  smooch_twitter_tier: 'sandbox'
+  smooch_twitter_env_name: 'checkmessage'
```

Comment:
Just to clarify more, those are the settings of a Twitter app used for Check Message, for the Twitter DMs integration

Reply:
are `sandbox` and `checkmessage` hardcoded values or they need to be set on the Twitter or Smooch side as well ?

Reply:
on Twitter side as well